common-good only installs with recent version of npm - how to fix "npm ERR! Invalid dependency type requested: alias"

Starting at commit 393f7ef08708da0a23bc9da5b9729fd6ab2a5841 "Update Common-Good to fix Windows problem" I was unable to run `npm install`.

`npm ERR! Invalid dependency type requested: alias`

I had npm 6.3.0 and node v10.8.0 on OSX 10.11.6.

Apparently a dependency somewhere within `common-good` is using "alias" in package.json, which is a yarn feature that only became supported by [npm 6.9.0 released in March 2019](https://npm.community/t/release-npm-6-9-0/5911).  Unfortunately the npm log didn't specify which package was causing the problem.

I updated npm using `npm install -g npm@latest`.  Now with npm 6.13.6, it all works.

Reporting this in case anyone else runs into it.

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
   },
   "devDependencies": {
     "changelog-version": "^1.0.1",
-    "common-good": "^1.1.18",
+    "common-good": "^1.1.19",
     "husky": "^3.0.5",
     "mkdirp": "^0.5.1",
     "nodemon": "^2.0.2",

